content/tutorials/token-authentication.textile

```diff
@@ -1568,7 +1593,7 @@ blang[swift].
     }
   ```
 
-  We will also update @getTokenRequest@ method so that it will use correct URL request:
+  You will also update the @getTokenRequest@ method so that it will use the correct URL request:
```

Comment:
"You need to" or retain "We" wouldn't make sense to say 'you will' as it sounds like we are kinda predicting what they'll do rather than guiding them to do something as mentioned above :)